Sources/SwiftProtobuf/Google_Protobuf_ListValue+Extensions.swift

```diff
@@ -44,14 +44,17 @@ extension Google_Protobuf_ListValue: _CustomJSONCodable {
       return
     }
     try decoder.scanner.skipRequiredArrayStart()
+    try decoder.scanner.incrementRecursionDepth()
```

Comment:
Probably worth a comment in this function to call out why manual recursion management is done for anyone looking at the code in the future.

Reply:
Done.